Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    intercom-rails (0.2.29)
+    intercom-rails (0.2.30)
```

Comment:
I don't understand why we're editing the `Gemfile.lock` directly, this seems like bad practice. But I see that you're not the first one to do this :(.